contracts/script/Deploy.s.sol

```diff
@@ -2,17 +2,22 @@
 pragma solidity ^0.8.19;
 
 import {Script, console} from "@forge-std/Script.sol";
+import {FROSTGroupId} from "@/libraries/FROSTGroupId.sol";
 import {FROSTCoordinator} from "@/FROSTCoordinator.sol";
+import {Consensus} from "@/Consensus.sol";
 
 contract DeployScript is Script {
-    function run() public returns (FROSTCoordinator) {
+    function run() public returns (FROSTCoordinator, Consensus) {
         vm.startBroadcast();
 
         FROSTCoordinator coordinator = new FROSTCoordinator{salt: bytes32(0)}();
+        console.log("FROSTCoordinator deployed to:", address(coordinator));
+
+        Consensus consensus = new Consensus{salt: bytes32(0)}(address(coordinator), FROSTGroupId.T.wrap(0xfa3efbc5dd215abad66149ad58d7869f90cf6d2c1bc472be0000000000000000));
```

Comment:
I imagine that this should, at some point, be read from `vm.getEnv`.

Reply:
yes, totally